src/subtasks/configure.js

```diff
@@ -6,21 +6,24 @@ const difference = require('lodash.difference');
 const yaml = require('js-yaml');
 const util = require('./../util');
 
-const updateStruct = (cwd, files, converter) => files.forEach((fileName) => {
-  const filePath = path.join(cwd, fileName);
-  const expected = converter[0](fs.readFileSync(path
-    .join(__dirname, '..', 'templates', 'merge', `dot${fileName}`), 'utf8'));
-  const actual = converter[0](fs.readFileSync(filePath, 'utf8'));
-  fs.writeFileSync(filePath, converter[1](defaults(actual, expected)).trim('\n'), 'utf8');
-  fs.appendFileSync(filePath, '\n');
-});
+const updateStruct = (cwd, files, converter) =>
+  files.forEach(fileName => {
+    const filePath = path.join(cwd, fileName);
+    const expected = converter[0](
+      fs.readFileSync(path.join(__dirname, '..', 'templates', 'merge', `dot${fileName}`), 'utf8')
+    );
```

Comment:
These changes I can agree with. The pythonic linebreaks make a lot of sense to me.

Reply:
Agreed. The gist is that with prettier there aren't a lot of options. What you see is what you get